### ISSUE TITLE
xgboost python does not display information even when silent = False

### ISSUE THREAD

**Issue by ganaz** [dmlc/xgboost#3639]
XGBoost is not printing any training information even when silent=False.

I am using below code :

`xgb1 = XGBClassifier(
        learning_rate =0.1,
        n_estimators=1000,
        max_depth=7,
        min_child_weight=1,
        gamma=0,
        subsample=0.8,
        colsample_bytree=0.8,
        objective= 'binary:logistic',
        nthread=31,
        scale_pos_weight=1,
        silent=False,
        seed=27)
`
`xgb1.fit(x_train, y_train,eval_metric='auc')`

Environment info:

Operating system : Ubuntu 16.04
Python Version : 3.6
Xgb verson : 0.72.1

And I am running the code in Jupyter .(Anaconda 3)

Thanks in advance.

**Comment by hcho3** [comment]
You should upgrade XGBoost to version 0.80 to print training information inside Jupyter notebook.

**Comment by ganaz** [comment]
Thanks for quick reply, I'll try upgrading xgboost version.
